lib/rust/profiler/flame-graph/src/lib.rs

```diff
@@ -163,7 +163,7 @@ impl<'p, Metadata> RungraphBuilder<'p, Metadata> {
                 let start = active.interval.start.into_ms();
                 let mut end = active.interval.end.map(|mark| mark.into_ms()).unwrap_or(f64::MAX);
                 // We want to show every wakeup of the task, even if it is momentary.
-                const DURATION_FLOOR_MS: f64 = 0.5;
+                const DURATION_FLOOR_MS: f64 = 2.0;
```

Comment:
I missed that in previous review – these docs should explain a little bit better what `DURATION_FLOOR_MS` is. The explanation "We want to show every wakeup of the task, even if it is momentary." is not clear IMO. If I understand correctly, this is a minimal "width" of a bar in the flame graph to make it easier visible, right?

Reply:
Yes, but it's in the run graph, which illustrates async relationships. (In the flame graph, short intervals don't matter.)